about/user_software.md

```diff
@@ -36,13 +40,47 @@ When spack packages are installed, they are accompanied by a modulefile. Therefo
 spack load package_name
 ```
 
-or 
+or
 
 ```shell
 module load package_name
 ```
 
-## Additional Software
 
-In general, we recommend users install additional software to their home directory
-and independently manage/maintain it. 
+## MPI
+> This section will discuss usage of MPI on Arjuna, and presumes familiarity with MPI.
+
+Arjuna provides MPI support via the [OpenMPI](https://www.open-mpi.org/) module (`module load openmpi/4.1.3`).
+Users should load this module if they are using MPI. The following prior installations are deprecated and not reccomended:
+
+- `module load openmpi/4.1.1`
+- MPI Installation at `/usr/local/mpirun`
+
+> `/usr/local/mpirun` is on the `PATH` by default. However it is **not supported**, and will be removed in the future.
+> Users are __strongly__ encouraged to use `module load openmpi/4.1.3` instead.
+
+Using legacy launchers (i.e. `mpirun` or `mpiexec`) is __not supported__ using `openmpi/4.1.3` and __not recommended__ on other mpi installations. Please use `srun` to launch mpi jobs. 
```

Comment:
What? Should this be a list?

Reply:
no, the sentence as written is exhaustive, therefore it should be i.e. the only things it excludes are mpirun and mpiexec, which it refers to as the "legacy launchers"